CMakeLists.txt

```diff
@@ -1,27 +1,28 @@
-cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
-project(psi4pasture LANGUAGES C CXX VERSION 0.0.1)
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+project(pasture LANGUAGES C CXX VERSION 0.0.1)
 
-# sortout transqt2/ccsort interdependence
 if(ENABLE_transqt2 OR ENABLE_ccsort)
   if(NOT (ENABLE_ccsort AND ENABLE_transqt2))
     message(STATUS  "ccsort and transqt2 enabled since interdependent")
     set(ENABLE_ccsort "ON")
     set(ENABLE_transqt2 "ON")
   endif()
 endif()
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 find_package(psi4 1.0 REQUIRED)
+include(psi4PluginCache)
```

Comment:
It does work in both cases although I don't know what precedence it takes over command line arguments. This is basically using the settings in `psi4PluginCache.cmake` as the default settings for the project.